packages/rxjs-firebase/src/operators/__tests__/documentExists.test.ts

```
@@ -0,0 +1,34 @@
+import { type DocumentSnapshot } from '@firebase/firestore'
```

ESLint

[error] Filename is not in kebab case. Rename it to `document-exists.test.ts`.

(unicorn/filename-case)